_posts/2024-06-03-geoserver-tutorials.md

```diff
@@ -0,0 +1,94 @@
+date: 2024-06-05
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### How to Implement Basic Security in Geoserver
+In this session, we want to discuss the Security section in GeoServer, Defining Users, Groups, and Roles, and Granting rights to created users. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/KCTGZJ2Trvw).
+
+[![](https://img.youtube.com/vi/KCTGZJ2Trvw/0.jpg)](https://www.youtube.com/watch?v=KCTGZJ2Trvw)
+
+## Introduction
+GeoServer has a robust security subsystem. Most of the security features are available through the Web Administration interface. In the Security panel, you can find links to set user properties and bind data to security rules. The basic idea is that you create users and roles, and then combine them with data to enable a specific set of access policies. You can also limit read and write access by role.
+
+## Defining users, groups, and roles
+Security in GeoServer is based on a role system where each role defines a specific set of functions. You can assign roles to users and groups; that is, assign functions to real people using your system. To ensure data security, you must identify who is accessing your layers and services.
+
+To organize your real users, GeoServer provides you with the user, group, and role concepts. With the first two, you can insert real people into the GeoServer security subsystem, and with roles, you can grant rights to real users.
+
+### User definition
+In GeoServer, a user is someone entitled to use the system; it may be another software or a real person. When you add a user to the security system, GeoServer stores a username, uniquely identifying the user, a password, and a set of key/value pairs to store general information about it. You can disable a user at any time, preventing him from using the system.
+
+### Group definition
+A group is a set of users. GeoServer stores a list of usernames belonging to the group and a group name, uniquely identifying the group.
+
+### Roles definition
+GeoServer roles are associated with performing certain tasks or accessing particular resources. Roles are assigned to users and groups, authorizing them to perform the actions associated with the role. So the GeoServer may have many users and it is difficult to assign a role to everyone, therefore, groups can be created and roles can be considered for each group according to the users.
```

Comment:
> So the GeoServer may have many...

This appears to rather fit into ### Group definition, don't you think?

And also the Note below?

Reply:
I fixed it.